### PR TITLE
Bifrost sizing

### PR DESCRIPTION
**Pull request by jahn96**
- [x] Horizontal scroll in chart chooser.
- [x] Change the size of the view when facet is added
- [x] Have the view fit to the graph content
    * Graph area in grid doesn't have a fixed height
    * To prevent the cut when the name of the categories are long
 
## Future Work
* Resizing the graph

### CODE REVIEW

**Comment by jahn96** [comment]
Resizing the graph did not work that well. I will just put it on hold and come back to it later, if necessary. Feel free to tackle this if you want.

**Review by Waidhoferj — CHANGES_REQUESTED** [review]
The new sizing logic doesn't seem to work as intended:
Screen Shot 2021-08-11 at 1 43 24 PM
Faceted graphs push the sidebar off of the screen so it is inaccessible.


https://user-images.githubusercontent.com/33008436/129100752-9da3e38f-9939-4d30-9572-aabaa8c1650e.mov

The ChartChooser side scroller has chart instances concealed offscreen because the widget is overflowing its bounds.

**Comment by jahn96** [comment]
The sizing was calculated based on only when the left sidebar is collapsed. The possible solution is to get the size of the sidebar on mount and calculate the width. Downside with this is that it doesn't change the width dynamically when the sidebar grows or shrinks.

**Comment by Waidhoferj** [comment]
Since CSS is giving us issues, we could listen for the `resize` event and get the with as state in the React widget, then set the width and the height of the chat chooser manually.

**Review by Waidhoferj — CHANGES_REQUESTED** [review]
When the columns collapse into rows, the side scroller doesn't take up the whole horizontal space, which makes is hard to view the entire width of a single graph. Maybe theres an issue in the resizing logic? We could just take the `boundingClientRect` of the widget container.


https://user-images.githubusercontent.com/33008436/129802072-d8f96fd2-aee4-4a72-ba2f-824a7ae42184.mov

File: src/components/Onboarding/ChartChooser.tsx [hunk 1]
```diff
@@ -40,17 +40,21 @@ const suggestedChartCss = (theme: BifrostTheme) => css`
   .chart-col {
     position: relative;
     max-height: 100%;
+    width: 100%;
     align-self: flex-start;
+    overflow: hidden;
   }
 
   .suggested-charts {
     height: 300px;
```
Comment (Waidhoferj) [inline src/components/Onboarding/ChartChooser.tsx:49]: There is still a vertical scroll on the suggested charts when the columns collapse to rows. To fix this, increase the height a bit.
```suggestion
    height: 315px;
```

Other than that looks great!

**Review by Waidhoferj — CHANGES_REQUESTED** [review]
One quick change

**Review by Waidhoferj — APPROVED** [review]
LGTM

**Pull request merged**